### ISSUE TITLE
Any running version?

### ISSUE THREAD

**Issue by mrrookes** [bokybanton/pgn-viewer#2]
Hi.
Is there any running version that I can have a look at?

**Comment by bokybanton** [comment]
There is not stable running version but you can check out the chessboard and chess.js plugins runing at https://ajedrez.bokyopinion.info :-P

**Comment by mrrookes** [comment]
:^( Still empty... Some players are finding in my database the lack of a search tool, and your have it. It would be nice to have it working.

**Comment by bokybanton** [comment]
https://ajedrez.bokyopinion.info/herramientas/database/ will work with your PGN files

**Comment by bokybanton** [comment]
Tell me if you can test it

**Comment by mrrookes** [comment]
Yes, it worked. I have to think what is the best way of combining your database and my viewer. Maybe just a link :).

**Comment by bokybanton** [comment]
OK, right now I will improve the viewer with common characteristics from other chess viewers, for example displaying a chess position by clicking on middle game move and of course, the stockfish.js plugin to analyse games on the fly from your web browser!!  @mrrookes you can see a link to your website here -> https://ajedrez.bokyopinion.info/enlaces/  it will be glad to see my link on your website 💯  👍 

PD: The database search was made using PHP and SQL tables, I needed to transform the PGN games into SQL rows to do the searches. By the way it is not necessary to convert PGN into tables but it was more easy for me using a PHP Script to parse the PGN games.

**Comment by mrrookes** [comment]
Noted! I am in the process of writing wome new css code. Too busy to add stuff :(
